tests/Unit/PointwiseFunctions/EquationsOfState/Test_Tabulated.cpp

```diff
@@ -0,0 +1,135 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <string>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"  // IWYU pragma: keep
+#include "IO/H5/ReadTabulated.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+namespace {
+
+void test_tabulated() noexcept {
```

Comment:
Given the seemingly random numbers, it would be good to explain where they come from. Are they meant to be random but close? Or did you take them from somewhere?

Reply:
For the 1D arrays they are the first and second numbers in the corresponding dataset and for the 3D arrays they are the first 2x2x2 block (so all elements with indices [0][0][0] -> [1][1][1]). I thought it would be better to have the indices in the sample file correspond to the actual indices in the full h5 file which is why I did not choose to test random numbers in the array.